### ISSUE TITLE
Improve contrast on widgets, update theme colors

### ISSUE THREAD

**Issue by NietGiftig** [t0mg/reacticz#12]
This is not a real request but merely a question to consider an enhancement.
Because of my (poor) eyesight i have some problems with certain colour schemes
White letters on a yellow background almost disappear for me, black letter are perfect to see (for me)

I do my own builds so it is possible to change some colours in the css files, no problem.

But Reacticz is at the moment of course in heavy movement with widgets adding or changing.
Would it be possible to make one css file with the basic colours in it and import these in the others.
Then I , and perhaps others, have an easier task to use some other colours for letters and background, and do not have to do this after every update
No need for extensive gui colour changes, just plain css changes manual in one file.

**Comment by t0mg** [comment]
Thanks, this is very important feedback. Ideally, you should not need to make any change to the project to be able to use the dashboard properly. This is my mistake if the widgets aren't readable enough. I can consider adding theming options (allowing to change the colors from within Reacticz, as part of the configuration), but I'd rather focus on having a default version that is as accessible as possible. 

Please have a look at these explorations: http://codepen.io/t0m/pen/OWMdEe and tell me which alteration (or combination of alterations) would work best in your opinion. Feel free to fork and explore yourself :)

**Comment by NietGiftig** [comment]
It's really not a "Fault", but more a concept from nature that every single person sees a color combination different from another.
That makes the world colorful. ;-)
[https://designschool.canva.com/blog/color-science/](url)
I personally like for this kind of applications soft pastel color tones, usually with black letters.
As soon as you change the letters to different colors some people have problems because combinations of colors can be problematic for the eye.
As far as I know is black on pastel colors for most people good visible.
But grey letters on some colors are becoming more difficult.

TL;DR 
For me pastel colors with black letters, normal thickness
Ivory  #FFFFF0
Aquamarine #7FFFD4
Cornsilk #FFF8DC  
Khaki #F0E68C
LightCyan #E0FFFF

**Comment by t0mg** [comment]
I did make a mistake when I didn't ensured that the interface provides proper contrast. While it is certainly true that we don't all perceive the colors the same way, [there are generic standards for the Web](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast) that when followed guarantee a decent level of legibility for _most_ people. 

So I will first address this issue by tweaking the default interface and making sure it is more readable, but I'll keep the strong colors because bold colors are trendy and I like that yellow :) Same with white or colored text, if the contrast is good I will keep using them.

However, I'll probably also make it possible to customize the colors (in one single place) so if you want to use only black text and pastels you'll also be able to do so much more easily.

**Comment by NietGiftig** [comment]
Thanks, you are my, yellow, (wo)man!!

**Comment by t0mg** [comment]
@NietGiftig please let me know if the new theme looks better for you. 

This commit introduces a theme selector in the Info view (last icon on the right in the menu). The new Theme.js file should let you create custom color schemes much more easily. If you create one with higher even contrast please do a pull request! Thanks

**Comment by NietGiftig** [comment]
!!! Thanks !!!
This is the cream on the cake.
I keep your favorite yellow with the black for now because it is readable for me!
I will most certainly make a color scheme with my favorite colors.
Thanks again.
I will also make a post in the Domoticz forum to alert other people to this feature
